app/scripts/update-problem.js

```diff
@@ -0,0 +1,45 @@
+// module level exports
+const opn = require('opn');
+const {runApp} = require('../server/app');
+
+const {
+  CONTENT,
+  CONTENTRC,
+  CONTENTRCNAME,
+  PATH,
+  TEMPLATES,
+} = require('../config');
+
+const {
+  fsMkdir,
+  fsRead,
+  fsStat,
+  fsWrite,
+  fsWriteIfExists,
+  mkdirp,
+} = require('../utils');
+
+/*
+ *  @function updateProblem
+ */
+const updateProblem = (argv) => {
+  
+  const uuid = argv._[1];
+  const problemDirName = `${CONTENT}/${uuid}`;
+
+  // UUID Error Checks
+  if (!uuid) { console.log("Error: Please select a UUID."); return; } 
+  fsStat(problemDirName)
+    .then(_ => { return runApp() })
```

Comment:
Just a quick stylistic note (don't have to make this change) but:

```
.then(_ => { return runApp() })
```

can be rewritten as:

```
.then(runApp)
```